app/src/componentLibrary/dropdown/dropdownOption/dropdownOption.scss

```diff
@@ -26,7 +26,7 @@
   cursor: pointer;
 }
 
-@mixin hover-state($backgroundColor, $color) {
+@mixin hover-state($backgroundColor, $borderColor, $color: $COLOR--almost-black) {
```

Comment:
I do not see, where we are using `borderColor` variable

Reply:
Me too
Nice catch)